guides/micronaut-intellij-idea-ide-setup/micronaut-intellij-idea-ide-setup.adoc

```diff
@@ -0,0 +1,82 @@
+This guide teaches you how to set up https://www.jetbrains.com/idea/[IntelliJ IDEA] to develop Micronaut applications.
+____
+IntelliJ IDEA – the Leading Java and Kotlin IDE
```

Comment:
:x: I would personally remove subjective comments like these. Although I could agree, I don't think it suits in a guide.

Reply:
removed via https://github.com/micronaut-projects/micronaut-guides/pull/1241/commits/10bdd501addc800a9e8f3142c92ccd382465c60e